files/scripts/hardencontainerpolicy-fix.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+# Tell build process to exit if there are any errors.
+set -oue pipefail
+
+sed -i 's/insecureAcceptAnything/reject/' /etc/containers/policy.json
+
+
+# Exception for build-container-installer to allow the ISO generation script to work
+# https://github.com/JasonN3/build-container-installer/issues/123
+yq -i -o=j '.transports.docker |=
+    {"ghcr.io/jasonn3": [
+        {
+          "type": "sigstoreSigned",
+          "keyPath": "/etc/pki/containers/build-container-installer.pub",
+          "signedIdentity": {
+            "type": "matchRepository"
+          }
+        }
+      ]
+    }
++ .' /etc/containers/policy.json
+
+yq -i -o=j '.transports.docker |=
+    {"ghcr.io/zelikos": [
+        {
+          "type": "sigstoreSigned",
+          "keyPath": "/etc/pki/containers/davincibox.pub",
+          "signedIdentity": {
+            "type": "matchRepository"
```

Comment:
this works, but please factor the yaml properly. leave common-scripts but remove the hardening script, then make two new yamls:

cosmic-scripts.yml and non-cosmic-scripts.yml

and put hardencontainerpolicy.sh in non-cosmic-scripts.yml and hardencosmiccontainerpolicy.sh in cosmic-scripts.yml

Reply:
Done. I left the common-scripts file for now. No need to change every build.
First lets fix the cosmic build. I can later swap common-scripts for non-cosmic-scripts in the other recipes.